src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
-'use strict';
+const populationElements = document.querySelectorAll('.population');
+let totalPopulation = 0;
 
-// write your code here
+populationElements.forEach((element) => {
+  totalPopulation += parseInt(element.textContent.replace(/,/g, ''), 10);
+});
+
+const averagePopulation = totalPopulation / populationElements.length;
+
+const totalPopulationElement = document.querySelector('.total-population');
+const averagePopulationElement = document.querySelector('.average-population');
+
+totalPopulationElement.textContent = totalPopulation.toLocaleString('en-Us');
+
+averagePopulationElement.textContent =
+  averagePopulation.toLocaleString('en-Us');
```

Comment:
The locale string should be 'en-US' instead of 'en-Us'. This is a minor typo but could lead to unexpected results.